src/amo/components/HomeHeroBanner/index.js

```diff
@@ -1,19 +1,47 @@
 /* @flow */
 import * as React from 'react';
 import { compose } from 'redux';
+import makeClassName from 'classnames';
 
 import translate from 'core/i18n/translate';
 import Hero from 'ui/components/Hero';
 import HeroSection from 'ui/components/HeroSection';
 import type { I18nType } from 'core/types/i18n';
+import tracking from 'core/tracking';
+import {
+  withExperiment,
+  type WithExperimentInjectedProps,
```

Comment:
Can we split this import into two imports? One regular `import` and one `import type`? We put all `import type` after the regular `import`.